frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -1,17 +1,18 @@
-import * as _ from 'lodash-es';
 import * as React from 'react';
 import * as classNames from 'classnames';
 import { History, Location } from 'history';
+import * as _ from 'lodash-es';
```

Comment:
Sorry I did not notice before. Lint does not seem to throw an error in this package, but `lodash-es` is a restricted import since webpack already uses `lodash-es`. can we change the import to `lodash`?

Reply:
As discussed in slack, the public package uses 213x lodash-es and only 14 times lodash. So I think we should not fix / change this individual import at the moment.

Reply:
okay. we can change this later if we sanitize according to lint later.